sql/core/src/test/resources/sql-tests/inputs/cte-nonlegacy.sql

```diff
@@ -0,0 +1,2 @@
+--SET spark.sql.legacy.ctePrecedence.enabled = false
+--IMPORT cte.sql
```

Comment:
we should do the same thing for `cte-legacy.sql`. This can be done in a followup.

Reply:
Copy that, will do it later.